urllib3/response.py

```diff
@@ -132,6 +132,7 @@ def __init__(self, body='', headers=None, status=0, version=0, reason=None,
         self._fp = None
         self._original_response = original_response
         self._fp_bytes_read = 0
+        self._msg = msg
```

Comment:
Is there a reason we need to use a property for this?

Reply:
msg has the headers in list from (headers has it in dict).
This is important for the Set-Cookie and  Set-Cookie2 headers that can are multiple-values-same-key headers that the headers by combining them into a single key lose the ability to parse them and set the value accordingly in the session header.

msg is also used in the _original_response context only by the stack which expects Response object of httplib2. A different solution might be to subclass this class and put msg only in the sub class and make _original_response instantiate that new sub class but this looks like a bit overrkill for the problem in hand.

Reply:
Sorry, I guess my question was not clear enough. Let me rephrase: Why can't we more simply do `self.msg = msg`?